src/screens/EventDetailsScreen/Header.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+import React from "react";
+import {
+  View,
+  Text,
+  StyleSheet,
+  TouchableOpacity,
+  StatusBar
+} from "react-native";
+import SafeAreaView from "react-native-safe-area-view";
+import { eventDetailsHeaderBgColor } from "../../constants/colors";
+
+type HeaderProps = {
+  onBackButtonPress: () => void
+};
+
+const Header = ({ onBackButtonPress }: HeaderProps) => (
+  <SafeAreaView style={styles.container}>
+    <StatusBar barStyle="default" animated />
+    <View style={styles.header}>
+      <TouchableOpacity onPress={onBackButtonPress}>
+        <Text>Back</Text>
```

Comment:
Should this text "Back" be extracted into `src/constants/text.js`?

Reply:
Maybe, but I'm currently just using it as a placeholder for an Icon. The word `back` won't be there in the final design.

Reply:
Ohhh, that makes sense. We really don't need to extract placeholder text.